imap_processing/cdf/config/imap_ultra_l1b_variable_attrs.yaml

```diff
@@ -26,113 +26,157 @@ x_front:
   CATDESC: x front position
   FIELDNAM: Event x-position on front foil
   LABLAXIS: x front position
-  # TODO: come back to format
-  UNITS: mm
+  VALIDMIN: -1e6
+  FORMAT: F12.6
+  UNITS: hundredths of a millimeter
```

Comment:
SPDF says the following about the UNITS attribute:

> is a character string (no more than 20 characters, but preferably 6 characters) representing the units of the variable

Is there any way to shorten these units? e.g. "m * 10^-5" or "mm / 100"